genny/auth/templates/actions/users_test.go.plush

```diff
@@ -9,21 +9,118 @@ func (as *ActionSuite) Test_Users_New() {
 	as.Equal(200, res.Code)
 }
 
-func (as *ActionSuite) Test_Users_Create() {
+func (as *ActionSuite) CreateTestUser(u *models.User, shouldFail bool) {
```

Comment:
Indentation is somewhat off in this file at several places, maybe you can fix it

Reply:
should be good now, please let me know if you notice something still off